src/api/handlers/projects/projects.py

```diff
@@ -34,6 +34,11 @@
 
 add_project_model = ns.schema_model('AddProject', add_project_schema)
 
+if os.environ['INFRABOX_HA_ENABLED'] == 'true':
```

Comment:
this is very error prone, because we need to do this check everywhere.

Reply:
I will move this condition in pyinfraboxutils